docs/root/configuration/http_filters/rbac_filter.rst

```diff
@@ -0,0 +1,32 @@
+.. _config_http_filters_rbac:
+
+Role Based Access Control (RBAC) Filter
+=======================================
+
+The RBAC filter is used to authorize actions (permissions) by identified downstream clients
+(principals). This is useful to explicitly manage callers to an application and protect it from
+unexpected or forbidden agents. The filter supports configuration with either a whitelist (ALLOW) or
```

Comment:
nit: hate to be that person, but can we change the language to be "safe list"/"block list" instead? Parathesis definitely help, but would like to change the language as well to be inclusive. thanks!

Reply:
"allow list" is another option